### ISSUE TITLE
Regression: Golden Axe: Beast Rider and Iron-Man crashes with (texture_cache.h:2564)

### ISSUE THREAD

**Issue by Emulator-Team-2** [RPCS3/rpcs3#7008]
I don't know which pull request regressed these games because they froze before the main menu from build (0.0.6-8603) to (0.0.7-8819).

And build (0.0.7-8819) already had an issue with texture cache fatal error.

Tested with the latest master build (0.0.7-9145).
Fatal error:
```
RSX: class std::runtime_error thrown: Verification failed: 
(in file C:\projects\rpcs3\rpcs3\Emu\RSX\Common\texture_cache.h:2564)
```
Log:
(Iron-Man) - [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/3856353/RPCS3.log.gz)
(Golden Axe: Beast Rider) - [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/3856364/RPCS3.log.gz)

**Comment by Realmantik** [comment]
Iron-man regressed after #6421, #6423.

**Comment by Emulator-Team-2** [comment]
Regression after build (0.0.6-8603) is already fixed by (0.0.7-8819).

**Comment by Realmantik** [comment]
With settings that can be seen in [this](https://www.youtube.com/watch?v=qn64qf-n_PE) video the game goes ingame. I used last build(9151).
